docker/build.sh

```diff
@@ -26,25 +27,28 @@ if [[ -f $BASE_SHA_FILE ]]; then
 fi
 
 
-export BASE_IMAGE_NAME="permutive-liberica-base"
-export IMAGE_NAME="permutive-jre-liberica"
+export BASE_IMAGE_NAME="$REMOTE_REPO/permutive-liberica-base"
+export IMAGE_NAME="$REMOTE_REPO/permutive-jre-liberica-experimental"
```

Comment:
**question:** Should this have the `experimental` removed now?

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org/) standard</sub>

Reply:
Yep, just forgot to remove...